Library/Formula/libmemcached.rb

```diff
@@ -15,7 +13,11 @@ class Libmemcached < Formula
   end
 
   option "with-sasl", "Build with sasl support"
+  option "with-static", "Build static library"
+  option "with-memaslap", "Build with load generation and benchmark tool support"
+  option "with-hsieh-hash", "Build with hsieh hashing support"
 
+  depends_on "libevent" # requires --enable-libmemcachedprotocol
```

Comment:
Does this have to be mandatory? Can it not be optional or recommended, or is the gain in functionality useful enough to merit the new mandatory dependency?